<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommandForm</class>
 <widget class="QWidget" name="CommandForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Formulaire de commande</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelCommandTitle">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Importer TDMS + Formulaire</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelCommand">
     <property name="text">
      <string>Commande: tdmsimport</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>580</width>
        <height>428</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="formContainer" native="true">
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="labelFichierTDMS">
            <property name="text">
             <string>Fichier TDMS *</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="lineEditFichierTDMS"/>
            </item>
            <item>
             <widget class="QPushButton" name="buttonParcourir">
              <property name="text">
               <string>Parcourir...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelBaseSQLite">
            <property name="text">
             <string>Base SQLite *</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="lineEditBaseSQLite"/>
            </item>
            <item>
             <widget class="QPushButton" name="buttonParcourir_2">
              <property name="text">
               <string>Parcourir...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelConfigLoader">
            <property name="text">
             <string>Config Loader (.conf)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLineEdit" name="lineEditConfigLoader"/>
            </item>
            <item>
             <widget class="QPushButton" name="buttonParcourir_3">
              <property name="text">
               <string>Parcourir...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelCoverLoad">
            <property name="text">
             <string>Cover Load</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="lineEditCoverLoad"/>
            </item>
            <item>
             <widget class="QPushButton" name="buttonParcourir_4">
              <property name="text">
               <string>Parcourir...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
<